### ISSUE TITLE
Number of Training Epochs

### ISSUE THREAD

**Issue by hshreeshail** [carolineec/EverybodyDanceNow#29]
The paper gives the number of epochs to train for each stage as follows:
5 epochs for global stage
30 epochs for local stage
5 epochs for FaceGAN

For each stage, the code divides the training epochs into two parts `niter` (epochs at starting learning rate) and `niter_decay` (epochs at decaying learning rate). For a given stage, how do we divide the total epochs (say 30 for the local stage) into these two parts. Equally? Some other ratio?
